src/_utils/_git_secret_tools.sh

```diff
@@ -29,7 +29,19 @@ fi
 : "${SECRETS_OCTAL_PERMS_COMMAND:="_os_based __get_octal_perms"}"
 : "${SECRETS_EPOCH_TO_DATE:="_os_based __epoch_to_date"}"
 
-# Temp Dir
+# _SECRETS_GPG_ARMOR is expected to be empty or '1'.
+# Empty means 'off', any other value means 'on'.
+# See: https://github.com/sobolevn/git-secret/pull/661
+# shellcheck disable=SC2153
+if [[ -n "$SECRETS_GPG_ARMOR" ]] && [[ "$SECRETS_GPG_ARMOR" -ne 0 ]]; then
```

Comment:
I'd prefer not to use a separate `_SECRETS_GPG_ARMOR` env var, and instead just set the correct param into $args at line 169 below using `SECRETS_GPG_ARMOR`
Seems like simpler logic and less variables

Reply:
👍